docs/faq/faq.md

```diff
@@ -0,0 +1,236 @@
+
+    SQLMesh stores state information in database tables. By default, it stores this information in the same [database/connection where your project models run](../reference/configuration.md#gateways). You can specify a [different database/connection](../reference/configuration.md#state-connection) if you would prefer to store state information somewhere else. 
+    
+    SQLMesh adds information to the state tables via transactions, and some databases like BigQuery are not optimized to execute transactions. Changing the state connection to another database like PostgreSQL can alleviate performance issues you may encounter due to state transactions.
+
+??? question "How do I get column-level lineage for my dbt project?"
+    SQLMesh can run dbt projects with its [dbt adapter](../integrations/dbt.md). After configuring the dbt project to work with SQLMesh, you can view the column-level lineage in the SQLMesh browser UI:
+
+    ![SQLMesh UI displaying column-level lineage for Sushi dbt example](./faq/ui-colum-lineage_sushi-dbt.png)
+
+??? question "Do I have to run Python models in my SQL engine?"
+    No! SQLMesh executes [Python models](../concepts/models/python_models.md) wherever SQLMesh is running, and there are no restrictions on what they can do as long as they return a Pandas or Spark DataFrame instance.
+
+    If the data is too large to fit in memory, the model can process it in batches or execute the transformations on an external Spark cluster.
+
+??? question "How do incremental models determine which dates to ingest?"
+    dbt uses the "most recent record" approach to determine which dates should be included in an incremental load. It works by querying the existing data for the most recent date it contains, then ingesting all records after that date from the source system in a single query.
+
+    SQLMesh uses the "intervals" approach instead. It divides time into disjoint intervals based on a model's `cron` parameter then records which intervals have ever been processed. It ingests source records from only unprocessed intervals. The intervals approach enables features like loading in batches.
+
+??? question "How do I run an append only model in SQLMesh?"
+    SQLMesh does not support append-only models as implemented in dbt. You can achieve a similar outcome by defining a time column and using an [incremental by time range](../concepts/models/model_kinds.md#incremental_by_time_range) model or specifying a unique key and using an [incremental by unique key](../concepts/models/model_kinds.md#incremental_by_unique_key) model.
+
+## Company
+
+??? question "How does Tobiko Data make money?"
+
+    - Model execution observability and monitoring tools (in development)
+    - Github Actions CI/CD bot (in development)
```

Comment:
So we have an open source CI/CD bot. Maybe say "Enterprise Github Actions CI/CD App" and then make a sub-bullet or something that says "Advanced version of open source CI/CD bot".